environment/directories.rst

```diff
@@ -50,6 +47,39 @@ You can use :ref:`filesystem-project` for larger datasets.
    If you also have an account on `lsst-dev <https://developer.lsst.io/services/lsst-dev.html>`_,
    you can access your Notebook Aspect home directory via :file:`$HOME/jhome/` from lsst-dev.
 
+Preinstalled subdirectories
+---------------------------
+
+Within the :file:`$HOME` directory a number of directories are currently pre-created for new users.
```

Comment:
```suggestion
The :file:`$HOME` directory contains several directories that are pre-created for you.
```

Reply:
I mostly like this, except for the last edit on the line.  I had wanted to emphasize that they are created on _account creation._  So we could do "... for you when your account is created."

However, one of the other points of this review is to get @athornton to tell us whether that's true and when they _really_ are created - e.g., perhaps the system checks for them on every login and re-creates them then?

Reply:
At leach login, if they are missing, they are re-created.

https://github.com/lsst-sqre/nublado/blob/38dca234e5fa392e7bc08d2df049d9bc94ecd701/jupyterlab/runlab.sh#L57